### PR TITLE
Task Design

### PR DESCRIPTION
**Pull request by Morten-Moeller**
Hi all, I took some time this weekend to finally do some more design on my app. You are welcome to have a look at the code.
There are a lot of files but almost only css was done.

### CODE REVIEW

File: src/components/PlayButton.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ import styled from 'styled-components/macro'
 import Button from './Button'
 
 const PlayButton = styled(Button)`
-  display: flex;
-  justify-content: center;
+  display: felx;
```
Comment (roscha80) [inline src/components/PlayButton.js]: I guess u mean flex ;)

File: src/pages/PlayPage.js [hunk 2]
```diff
@@ -69,10 +90,10 @@ export default function PlayPage({
         </>
       )}
       {!isCounter && (
-        <Wrapper>
+        <WrapperEndGmae>
```
Comment (roscha80) [inline src/pages/PlayPage.js]: I guess u mean WrapperEndGame ;)

File: src/pages/PlayPage.js [hunk 3]
```diff
@@ -69,10 +90,10 @@ export default function PlayPage({
...
           Game ended! You got {playerData.score} points.
           <Button onClick={onEndGame}>End game</Button>
-        </Wrapper>
+        </WrapperEndGmae>
```
Comment (roscha80) [inline src/pages/PlayPage.js]: Again,
I guess u mean WrapperEndGame ;)

File: src/pages/PlayPage.js [hunk 4]
```diff
@@ -122,3 +155,9 @@ const Wrapper = styled.div`
     width: 70%;
   }
 `
+
+const WrapperEndGmae = styled.section`
```
Comment (roscha80) [inline src/pages/PlayPage.js]: And again,
I guess u mean WrapperEndGame ;)

**Review by roscha80 — COMMENTED** [review]
Boah, really nice design. I like it!

**Comment by Morten-Moeller** [comment]
Thank you Rob, i changed everything you mentioned.

File: src/pages/PlayPage.js [hunk 5]
```diff
@@ -35,11 +37,30 @@ export default function PlayPage({
...
+                      d="M94.735,41.928,16.161,1.285C9.777-2.016,0,1.187,0,9.351V90.617c0,7.324,9.085,11.738,16.161,8.066L94.735,58.06c7.009-3.613,7.031-12.519,0-16.132Z"
+                      transform="translate(0 -0.002)"
+                      fill="none"
+                      stroke="#ed67bf"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      strokeWidth="2"
+                    />
+                  </g>
+                </svg>
```
Comment (macfrei) [inline src/pages/PlayPage.js:58]: Is this the same `svg` you have in `/assets/play.svg`? If so, why are you not using that one? Also You should probably add an aria-label or you could add a `title` and `desc` to your inline svg. Have a look here: https://css-tricks.com/accessible-svgs/#2-inline-svg

**Pull request merged**